src/tv2-common/helpers/graphics/internal/index.ts

```diff
@@ -26,8 +20,8 @@ export function CreateInternalGraphic(
 	}
 
 	if (adlib) {
-		internalGraphic.createAdlibTargetingOVL(context, actions, adlibPieces)
-		internalGraphic.createAdlib(context, actions, adlibPieces)
+		internalGraphic.createCommentatorAdlib(adlibPieces)
```

Comment:
Follow up to the sending an array as a parameter instead of returning the newly created instance comment.

Here the code would then be:
```
if (adlib) {
  adlibPieces.push(internalGraphic.createCommentatorAdlib())
  adlibPieces.push(internalGraphic.createAdlib())
} else {
  pieces.push(internalGraphic.createPiece())
}
```

It seems small, but it has a huge impact on the overall codebase in the long run